### PR TITLE
DFPL-54 Child Assessment Order

### PR DESCRIPTION
**Pull request by so99y**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFPL-54

### Change description ###
Digitalise child assessment orders.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json [hunk 1]
```diff
@@ -3771,5 +3771,103 @@
...
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "manageOrdersChildThirdContactIfKeepAwayFromHome",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "manageOrdersIsCostOrderExist",
```
Comment (jamesrferguson1) [inline ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json:3862]: maybe manageOrdersDoesCostOrderExist

File: ccd-definition/FixedLists/CareSupervision/ManageOrdersChildAssessmentType.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "DisplayOrder": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ManageOrdersChildAssessmentType",
+    "ListElementCode": "PSYCHIATRIC_ASSESSMENT",
+    "ListElement": "Psychiatric Assessment",
+    "DisplayOrder": 2
+  }
+]
```
Comment (jamesrferguson1) [inline ccd-definition/FixedLists/CareSupervision/ManageOrdersChildAssessmentType.json:16]: warning about missing new line here

File: service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/Order.java [hunk 3]
```diff
@@ -10,7 +10,48 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
-import static uk.gov.hmcts.reform.fpl.model.order.OrderQuestionBlock.*;
+import static uk.gov.hmcts.reform.fpl.model.order.OrderQuestionBlock.APPOINTED_GUARDIAN;
```
Comment (jamesrferguson1) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/Order.java:13]: interesting it was ok as .* import previously

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/generator/C39ChildAssessmentOrderParameterGeneratorTest.java [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+        }
+
+        return C39ChildAssessmentOrderDocmosisParameters.builder()
+            .orderTitle(C39_CHILD_ASSESSMENT_ORDER.getTitle())
+            .childrenAct(C39_CHILD_ASSESSMENT_ORDER.getChildrenAct())
+            .orderByConsent("By consent")
+            .orderDetails(String.format(orderDetails, assessmentType.getTitle(), duration, duration > 1 ? "s" : ""))
+            .build();
+    }
+}
```
Comment (jamesrferguson1) [inline service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/generator/C39ChildAssessmentOrderParameterGeneratorTest.java:179]: new line :)

**Review by jamesrferguson1 — DISMISSED** [review]

**Review by AlistairEdwinOsborne — APPROVED** [review]
everything past Jame's approval looks good!

**Pull request merged**